src/@next/Input/InputStyle.ts

```diff
@@ -44,6 +44,7 @@ export const StyledContainer = styled.div<InputProps & PreffixSuffixWidthProps>`
     border: 1px solid ${Neutral.B85};
     background: ${Neutral.B95};
     color: ${Neutral.B85};
+    cursor: pointer;
```

Comment:
sorry, this should be not allowed instead of a pointer

Reply:
OMG lol, thanks!